src/components/Nav/Navigation.tsx

```diff
@@ -105,15 +107,21 @@ class Navigation extends React.Component<PropsType, StateType> {
           <VerticalNav.Item title={istioRulesTitle} iconClass="fa pficon-migration" onClick={this.navigateTo} />
           <VerticalNav.Item title={servicesJaeger} iconClass="fa fa-paw" onClick={this.navigateTo} />
         </VerticalNav>
-        <Switch>
+        <SwitchErrorBoundary
+          fallBackComponent={() => (
+            <div className="container-fluid container-pf-nav-pf-vertical">
```

Comment:
Please use `PfContainerNavVertical` instead

Reply:
Right, I always forget about that :), thanks.